src/praisonai/praisonai/cli/configuration/resolver.py

```
@@ -0,0 +1,543 @@
+            
+            if path.suffix in (".yaml", ".yml"):
+                return yaml.safe_load(content) or {}
+            elif path.suffix == ".toml":
+                return toml.loads(content)
+            elif path.suffix == ".json":
+                return json.loads(content)
+            else:
+                # Try to detect format
+                try:
+                    return yaml.safe_load(content) or {}
+                except:
```

Ruff (0.15.17)

[error] Do not use bare `except`

(E722)